client/src/components/Modal.js

```diff
@@ -9,14 +9,16 @@ const Modal = (props) => {
         className="ui standard modal visible active"
       >
         <div
+          id="white-text"
```

Comment:
Should these just be classes? I thought there could only be a single element with a particular id.

Reply:
I tried classes but for some reason having multiple classes does not work. For example the Modal.js, I tried to have header white-text (with .white-text {colour: white} in css) it would not inherit it for some reason so I used ID instead and it seemed to work